client/src/Components/CryptoList.jsx

```diff
@@ -0,0 +1,82 @@
+import { useState, useEffect } from "react";
+
+function CryptoList() {
+  const [cryptos, setCryptos] = useState([]);
+  useEffect(() => {
+    fetch(
+      "https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=100&page=1&sparkline=false&locale=en"
+    )
+      .then((response) => response.json())
+      .then((data) => setCryptos(data))
+      .catch((error) => console.error(error));
+  }, []);
+  console.log(cryptos);
+
+  return (
+    <div className="overflow-x-auto">
+      <table className="table" style={{ width: "1400px" }}>
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Price</th>
+            <th>Cap</th>
+            <th>24hr high</th>
+          </tr>
+        </thead>
+        <tbody>
+          {cryptos.map((crypto) => {
+            return (
+              <tr>

```

Comment:
could you change this to `<tr key={crypto.id}>` ?
it seems like react requires you to have a key attribute